interface/themes/core/patient/demographics.scss

```diff
@@ -34,3 +35,116 @@
     }
   }
 }
+
+
+#DEM table {
+  width: 100%;
+}
+
+#DEM .label_custom {
+  text-align: right;
+}
+
+#DEM .data {
+  padding-left: 8px;
+  padding-right: 15px;
+}
+
+
+/* Demographics - Previous Addresses - address_list_display.php */
+
+#DEM .table_text_addresses {
+  border-bottom: 2px solid #e9ecef;
+  margin-top: 1em;
+}
+
+#DEM .text_addresses.header {
+  background: #e9ecef;
```

Comment:
colors in scss should use theme color override variables.

Reply:
@sjpadgett
Yes, I should have used the variable $gray-400 in the scss code.

However, I now moved this to inline per your and Brady's suggestion.  Bootstrap has limited colors, and none were appropriate.  

gray-400 does not work in css.  How do I get named colors?  Do I need to add scss code to add named colors of gray-100 to gray-900?  Can I?  How and where do I do this?  

I also tried var(--gray-400) in the css styling but this did not work.  Why not?  From what I read, it should have.

Reply:
This is defining the var to use elsewhere.